frontend/packages/client/src/components/ProposalCreate/StepTwo.js

```diff
@@ -47,7 +47,10 @@ const StepTwo = ({ stepData, setStepValid, onDataChange }) => {
 
     // push now if date is today and not already in time interval
     if (cutOffDate) {
-      const nowDate = new Date();
+      const nowDate =
+        process.env.APP_ENV === 'PRODUCTION'
+          ? new Date(Date.now() + 60 * 60 * 1000) // delay by an hour in prod env
+          : new Date();
       nowDate.setSeconds(0);
```

Comment:
@fersan1985 I've checked on this file https://github.com/DapperCollectives/CAST/blob/ca706f83d8922e5a036e93816f24b3ff999cb8b1/frontend/Dockerfile.vars#L19-L20 and APP_ENV is not set in the docker build process. I do see NODE_ENV I think that's we should use instead.

Reply:
added APP_ENV on Dockerfile.stage file

Reply:
App env added https://github.com/DapperCollectives/CAST/blob/c96756d62fee476ebba3e9a4d142329cee977626/frontend/Dockerfile#L12-L13